tests/app/helpers/test_url_safe_serializer.py

```diff
@@ -0,0 +1,68 @@
+from datetime import datetime, timedelta
+
+from dateutil.tz import tzutc
+from flask import session as cookie_session
+from itsdangerous import BadSignature
+from mock import patch
+
+from app.data_models.session_data import SessionData
+from app.data_models.session_store import SessionStore
+from app.helpers import url_safe_serializer
+from app.settings import EQ_SESSION_ID
+from tests.app.app_context_test_case import AppContextTestCase
+
+
+class TestUrlSafeSerializer(AppContextTestCase):
+    def setUp(self):
+        super().setUp()
+        self.session_data = SessionData(
+            tx_id="tx_id",
+            schema_name="some_schema_name",
+            response_id="response_id",
+            period_str="period_str",
+            language_code=None,
+            launch_language_code=None,
+            survey_url=None,
+            ru_name="ru_name",
+            ru_ref="ru_ref",
+            case_id="case_id",
+            questionnaire_id="questionnaire_id",
+        )
+        self.session_store = SessionStore("user_ik", "pepper", "eq_session_id")
+        self.expires_at = datetime.now(tzutc()) + timedelta(seconds=5)
+
+    def test_safe_serializer_deserializes(self):
+        test_string = "test-string"
+
+        with self.app_request_context("/status"):
+            with patch(
+                "app.authentication.authenticator.get_session_store",
+                return_value=self.session_store,
+            ):
+                self.session_store.create(
+                    "eq-session-id", "user_id", self.session_data, self.expires_at
+                )
+                cookie_session[EQ_SESSION_ID] = "eq-session-id"
```

Comment:
I think it's a bit confusing to use the value of `EQ_SESSION_ID` as the session id.

Reply:
Fixed - now using uuid4 strings